starboard/extension/BUILD.gn

```diff
@@ -29,3 +29,13 @@ target(gtest_target_type, "extension_test") {
     deps += cobalt_platform_dependencies
   }
 }
+
+target(gtest_target_type, "setstring_test") {
```

Comment:
The target itself here looks fine. 

But if you want to make it part of the CI / automatic test runs in this branch, you'll also have to add this new target into `GetTestTargets` either in `cobalt/build/cobalt_configuration.py` or in the platform specific equivalent.

Reply:
In current implementation, this test is an integration test, because involves a host and a target parts. Hence, it cannot be a part of the CI / automatic test. This PR contains the target part.

Reply:
Lets describe this intent in the comment above the test itself - it's not obvious from the code. Other than that lgtm.